.eslintignore

```diff
@@ -1,7 +1,8 @@
-node_modules
+**/node_modules
```

Comment:
Docs say ESLint ignores `node_modules` by default, but it doesn't seem to ignore `node_modules` in subfolders.